components/interactive_dialog/dialog_element.jsx

```diff
@@ -166,6 +168,17 @@ export default class DialogElement extends React.PureComponent {
                     onChange={onChange}
                 />
             );
+        } else if (type === 'radio') {
+            return (
+                <RadioSetting
+                    id={name}
+                    label={displayNameContent}
+                    helpText={helpTextContent}
+                    values={options}
+                    value={value || options[0].value}
```

Comment:
If no default value is included in the payload, this renders 1st radio button as automatically selected.  I believe, we should leave it as no option selected similar to how other options work.

Should this be?
```suggestion
                    value={value}
```

Reply:
@saturninoabril The ticket specifically asks for the first element of the options:
> The default value is the first element of the list.

Reply:
Hmm, okay then looks good to me.